smack-omemo/src/main/java/org/jivesoftware/smackx/omemo/provider/OmemoBundleVAxolotlProvider.java

```diff
@@ -96,6 +97,11 @@ else if (inPreKeys && name.equals(PRE_KEY_PUB)) {
                     break;
             }
         }
+
+        if (preKeys.isEmpty()) {
+            throw new XmlPullParserException("OMEMO Bundle MUST contain at least one prekey.");
```

Comment:
This should be a SmackException as it is not thrown from within the XML pull parser.

Reply:
I noticed, that this change might be somewhat obsolete, as [this line](https://github.com/igniterealtime/Smack/blob/master/smack-omemo/src/main/java/org/jivesoftware/smackx/omemo/util/OmemoKeyUtil.java#L126) already makes sure that we do not use empty bundles. Still, it might be good to fail early on, right?